docs/sources/mimir/references/http-api/index.md

```diff
@@ -566,8 +566,17 @@ GET,POST <prometheus-http-prefix>/api/v1/cardinality/label_values
 Returns realtime label values cardinality associated to request param `label_names[]` across all ingesters, for the authenticated tenant, in `JSON` format.
 It returns the series count per label value associated to request param `label_names[]`.
 
-As far as this endpoint generates cardinality report using only values from currently opened TSDBs in ingesters, two subsequent calls may return completely different results, if ingester did a block
-cutting between the calls.
+Two methods of counting are available, `inmemory` and `active`. This choice is controlled via the `count_method` parameter.
+
+The `inmemory` method counts the number of series in currently opened TSDBs in Mimir's ingesters.
+Two subsequent calls may return completely different results if an ingester cut a block between calls.
+This method of counting is most useful for understanding ingester memory usage.
+
+The `active` method also counts series in currently opened TSDBs in Mimir's ingesters, but filters out series that have not received a sample within a configurable duration of time. 
+Use the `ingester.active-series-metrics-idle-timeout` parameter to configure this duration.
+This method of counting is most useful for understanding which label values are represented in the samples ingested by Mimir in the last `ingester.active-series-metrics-idle-timeout`.
+Two subsequent calls will likely return similar results, as this window of time is not related to the block cutting on ingesters.
```

Comment:
```suggestion
Two subsequent calls will likely return similar results, because this window of time is not related to the block cutting on ingesters.
```